### DISCUSSION TITLE
There was no property ID specified. You must provide a valid property ID to execute queries on Google Analytics.

### DISCUSSION THREAD

**Discussion by MerajCode** [freekmurze/laravel-analytics#495] [Q&A] [answered]
I am using GA4 analytics, laravel-analytics 5.0 and Laravel 10 and I have also entered property id correctly but still this problem is coming what to do

**Comment by jerryklimcik** [comment, marked as answer]
Hi @Meraj81034 , I had the same problem (with v5.0, laravel 10) and the problem was with `config/analytics.php` where I had a key `view_id` instead of `property_id`. Probably because of some old version when I upgraded and didn't notice this change. Hopefully it will help you to solve this problem 😏 

And in the worst case you can always debug you problem, just start here `vendor/spatie/laravel-analytics/src/Exceptions/InvalidConfiguration.php:11` 😃
